src/Kovan-DssSpell.sol

```diff
@@ -29,44 +26,31 @@ contract SpellAction {
     //  against the current release list at:
     //     https://changelog.makerdao.com/releases/kovan/active/contracts.json
 
-    ChainlogAbstract constant CHANGELOG = ChainlogAbstract(0xdA0Ab1e0017DEbCd72Be8599041a2aa3bA7e740F);
+    ChainlogAbstract constant CHANGELOG =
+        ChainlogAbstract(0xdA0Ab1e0017DEbCd72Be8599041a2aa3bA7e740F);
 
-    address constant MCD_ADM            = 0x27E0c9567729Ea6e3241DE74B3dE499b7ddd3fe6;
-    address constant VOTE_PROXY_FACTORY = 0x1400798AA746457E467A1eb9b3F3f72C25314429;
+    address constant MCD_DC_IAM = 0x0000000000000000000000000000000000000000;
 
     function execute() external {
-        address MCD_PAUSE   = CHANGELOG.getAddress("MCD_PAUSE");
-        address FLIPPER_MOM = CHANGELOG.getAddress("FLIPPER_MOM");
-        address OSM_MOM     = CHANGELOG.getAddress("OSM_MOM");
+        address MCD_VAT = CHANGELOG.getAddress("MCD_VAT");
 
-        // Change MCD_ADM address in the changelog (Chief)
-        CHANGELOG.setAddress("MCD_ADM", MCD_ADM);
+        // Give permissions to the MCD_DC_IAM to file() the vat
+        VatAbstract(MCD_VAT).rely(MCD_DC_IAM);
 
-        // Add VOTE_PROXY_FACTORY to the changelog (previous one was missing)
-        CHANGELOG.setAddress("VOTE_PROXY_FACTORY", VOTE_PROXY_FACTORY);
+        // Add MCD_DC_IAM to the changelog
+        CHANGELOG.setAddress("MCD_DC_IAM", MCD_DC_IAM);
 
         // Bump version
-        CHANGELOG.setVersion("1.2.0");
-
-        // Set new Chief in the Pause
-        DSPauseAbstract(MCD_PAUSE).setAuthority(MCD_ADM);
-
-        // Set new Chief in the FlipperMom
-        FlipperMomAbstract(FLIPPER_MOM).setAuthority(MCD_ADM);
-
-        // Set new Chief in the OsmMom
-        OsmMomAbstract(OSM_MOM).setAuthority(MCD_ADM);
+        CHANGELOG.setVersion("1.2.1");
```

Comment:
renBTC and UNI will likely be going in to 1.2.2 on mainnet. Do we want to try to keep these synced?

Reply:
I think everything can happen atomically in the spell, so there should only be one version bump.  The real question is, does adding this module constitute a `1.3.0` bump?

Reply:
I think we should do `1.2.1` this version as we are not replacing any existing contract.
So what exactly is going in the next mainnet spell? I guess that's what we should mark as `1.2.1` and add to this PR.

Reply:
I second `1.2.1` here